sys/linux/vmw_vmci.txt.const

```diff
@@ -0,0 +1,46 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
+arches = 386, amd64, arm, arm64, riscv64, s390x
```

Comment:
We now have 8 arches. If you don't have compilers installed, you may try syz-env:
https://github.com/google/syzkaller/blob/master/docs/contributing.md#using-syz-env
`syz-env make extract SOURCEDIR=~/linux` should give all 8 arches.

Reply:
Done